src/app/virtualmachines/vmOverview.component.ts

```diff
@@ -415,9 +420,13 @@ export class VmOverviewComponent implements OnInit, OnDestroy {
 		this.children.forEach(
 			(child: VmCardComponent) => {
 				if (child.is_checked) {
+					if (this.user_elixir_id !== child.vm.elixir_id) {
```

Comment:
If you select some machines and click  "delete selected machines" -> close -> click again : the machines will always be added again (so you can check if they are already in the array or clear the array before adding again)

![grafik](https://user-images.githubusercontent.com/29981967/135997513-83fb48f5-90ee-42b3-9e2b-7a02dc7a6f2d.png)